CondFormats/HcalObjects/interface/HcalLutMetadatum.h

```diff
@@ -16,23 +16,23 @@ class HcalLutMetadatum
     HcalLutMetadatum(unsigned long fid,
 		     float rCalib,
 		     uint8_t lutGranularity,
-		     uint8_t outputLutThreshold):
+		     uint8_t LutThreshold):
       mId(fid), 
       mRCalib(rCalib),
       mLutGranularity(lutGranularity),
-      mOutputLutThreshold(outputLutThreshold) {}
+      mLutThreshold(LutThreshold) {}
 
   uint32_t rawId() const {return mId;}
 
   float   getRCalib() const {return mRCalib;}
   uint8_t getLutGranularity() const {return mLutGranularity;}
-  uint8_t getOutputLutThreshold() const {return mOutputLutThreshold;}
+  uint8_t getLutThreshold() const {return mLutThreshold;}
 
  private:
   uint32_t mId;
   float    mRCalib;
   uint8_t  mLutGranularity;
-  uint8_t  mOutputLutThreshold;
+  uint8_t  mLutThreshold;
```

Comment:
@aminghiasi
here you are changing a data member name for the type HcalLutMetadatum, which is involved in persistent data for the payloads of type HcalLutMetadata.
The result of this is, the old version code won't be able to read new payloads written with the changed version, and in turn this one won't be able to read existing payloads...
Did you do any of these tests?